src/common/transformers/ebayui-attribute/index.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 const COMPONENT_DIR = path.join(__dirname, '../../../components');
 const COMPONENT_FILES = fs
     .readdirSync(COMPONENT_DIR)
-    .map(entry => path.join(COMPONENT_DIR, entry, 'template.marko'));
+    .map(entry => path.join(COMPONENT_DIR, entry, 'index.marko'));
```

Comment:
Does this cause any other issues for any other components? We still have like 6 more components to do so I just want to make sure that this doesn't break anything

Reply:
Ultimately we will have to change this, once we migrate all of them. 🤔 
Or do you want to change this at the end?